src/components/Filters/MultiFilter.tsx

```diff
@@ -47,7 +47,7 @@ class MultiFilter<O, V extends Value> extends BaseFilter<V[], MultiFilterProps<O
         {...props}
         compact={!vertical}
         label={this.label}
-        values={value || []}
+        values={value || defaultValue || []}
```

Comment:
Hm, I don't think this is right, b/c the intent of `defaultValue` was to be the initial value for the filter, and not necessarily "what the field must always be".

I.e. a valid flow would be:

* You have a default filter that defaults to `projects=mine`
* You clear the filter to be just `projects=undefined`

This would be put back `mine`, each time you tried to clear the filter.

Reply:
Good point, in that case, we could use the `tick` trick to keep track of the initial render and subsequent renders. WDYT? Did @bdow and yourself have another implementation in mind? This is needed for the weeklyUpdates work.

Reply:
Naive question, but why do you think this is not working? FWIW usually when I think that `defaultValue` is not working, it's because `usePersistedFilter` is pulling things from session storage.

Specifically, this is the code that handles "just for the very first filter, use default values":

```
  const defaultFilter = useMemo(
    () =>
      Object.fromEntries(
        safeEntries(filterDefs)
          .filter(([key, def]) => def(key as string).defaultValue !== undefined)
          .map(([key, def]) => [key, def(key as string).defaultValue]),
      ),
    [filterDefs],
  );
```

In `usePersistedFilter`. We can pull that out into like a `createDefaultFilter(filterDefs)` if the weekly updates page needs to call it directly i.e. from a custom `useMemo` if it doesn't want to use `usePersistedFilter`.